com.unity.ml-agents/Tests/Editor/MLAgentsEditModeTest.cs

```diff
@@ -438,6 +440,7 @@ public void TestAgent()
                 if (i == 2)
                 {
                     agent1.LazyInitialize();
```

Comment:
Similar comment to the other test. I think it's cleared to check the number of calls right around the point that they change.